### DISCUSSION TITLE
Welcome to cufsm-git Discussions!

### DISCUSSION THREAD

**Discussion by coldformed** [thinwalled/cufsm-git#23] [Announcements]
As we experiment with GitHub, also turning on discussions here inside the cufsm-git repository.

Still finding the right balance between issues and discussions, but for now maybe will keep issues related to items we intend to do a pull request on and make a fix. Discussions might relate to bigger picture ideas.

In general need to decide if cufsm is a library of buckling tools, or is it really just the Matlab GUI. For now, it is growing in both directions, will see how that goes.

**Comment by curiousnick** [comment]
Hi, I'm a new user to Cold-formed steel as part of my master's coursework and new to CUFSM package,utilising windows 10 installation. 
Could you guide me how to get the most out of the package by referring to any tutorials or workshops you have planned or documented online?

**Reply by akchurinda** [reply]
Hi @curiousnick,

You can find some of the older tutorials that are still useful on the CUFSM's official [website](https://www.ce.jhu.edu/cufsm/), which also provides some other information about the package. I also have a dedicated [playlist](https://youtube.com/playlist?list=PLuBOE4IEdKvysk2vZXCfS232grVXLCnVf&si=dGV7lFuVgH4NuaNQ) for all tutorials posted on YouTube.

**Comment by Heyuhanshixioagou** [comment]
Hello! I'm sorry to disturb you in your busy schedule.
I would like to ask you a technical question about the CUFSMV5.04 software. In particular, I would like to know how to export the section shape data calculated by the software, especially to obtain the position coordinate information of the key points after buckling.
My current problem: trying to use the built-in export function didn't work. So Here's the thing:
1. Is there a specific export procedure that needs attention?
2. Where is the default storage path for the software result file?
So I can try to write a program to extract the Data I need. If you can provide relevant guidance, I would be grateful!
Sincerely,
attention

**Reply by coldformed** [reply]
Are you a matlab user or are you trying to use the compiled exe? We only provide direct access to this type of output from matlab at this time. You could likely hack the 2D plots since they are vector based but that would be a huge workaround vs. just accessing the coordinates directly in matlab.

**Reply by Heyuhanshixioagou** [reply]
Hello,After performing calculations using MATLAB, I saved a .mat file. By using the who command, I found that the file contains a variable named 'shape'. I am unsure whether this variable represents the vector coordinates of the cross-sectional buckling deformation. However, I find it a bit difficult to understand, as the array of this 'shape' variable is a very large matrix, and I do not know how to extract the vector information from it.
